### PR TITLE
Add support JPRS whois

### PR DESCRIPTION
**Pull request by nekok500**
Because the response from JPRS whois (.jp whois server) is unusual and does not match existing patterns, i made some changes.

Normally, the resopnse would be in English.
```
$ LANG=C whois jprs.jp
...
[Created on]                    2001/02/02
[Expires on]                    2025/02/28
[Status]                        Active
[Lock Status]                   DomainTransferLocked
[Lock Status]                   AgentChangeLocked
[Last Updated]                  2024/04/25 16:32:36 (JST)
...
```

whoisjs.com doesn't seem to be the case.
```
$ curl https://whoisjs.com/api/v1/jprs.jp | jq ".raw" --raw-output
...
[登録年月日]                    2001/02/02
[有効期限]                      2025/02/28
[状態]                          Active
[ロック状態]                    DomainTransferLocked
[ロック状態]                    AgentChangeLocked
[最終更新]                      2024/04/25 16:32:36 (JST)
...
```

So I added a new pattern.

### CODE REVIEW

File: package-lock.json
Comment (MattIPv4) [inline package-lock.json:1]: Please revert the changes to this file

Comment (nekok500) [reply package-lock.json:1]: Ok, I reverted it.

**Review by MattIPv4 — APPROVED** [review]
